framework/tester.py

```diff
@@ -0,0 +1,163 @@
+import unittest
+import time
+
+from helpers import tempesta, control, stateful, tf_cfg
+
+from . import wrk_client, nginx_server
+from . import deproxy_client, deproxy_server, deproxy_manager
+from templates import fill_template
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2018 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+class TempestaTest(unittest.TestCase):
+    """ Test for tempesta """
```

Comment:
The comment is obvious and useless. Despite this, the class seems the corner stone for the whole framework - if a one wants to write a new test, then they must start for the class and inherit the new test class from it. So please write a good comment here or in README.md explaining how to create a new test, which methods and properties must be defined and called and so on.